build-image/Dockerfile

```diff
@@ -10,45 +10,48 @@ RUN apt-get install -y nodejs && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 # and viceversa.
 RUN npm install -g postcss-cli@5.0.1 autoprefixer@9.8.5
 
-ENV HUGO_VERSION=v0.72.0
-RUN git clone https://github.com/gohugoio/hugo.git --branch ${HUGO_VERSION} --depth 1 && \
-	cd hugo  && go install --tags extended && cd ../ && \
-	rm -rf hugo/ && rm -rf /go/pkg /go/src
-ENV SHFMT_VERSION=3.1.0
-RUN curl -fsSLo shfmt https://github.com/mvdan/sh/releases/download/v${SHFMT_VERSION}/shfmt_v${SHFMT_VERSION}_linux_amd64 && \
-	echo "cb91ea08a075a2f96b5230f09b4e211b7c108b1c97603caceb48d117d2ac5508  shfmt" | sha256sum -c && \
+ENV HUGO_VERSION=0.72.0
+RUN GOARCH=$(go env GOARCH) && echo $GOARCH && \
+	if [ "$GOARCH"="amd64" ]; then \
+		URL=https://github.com/gohugoio/hugo/releases/download/v${HUGO_VERSION}/hugo_${HUGO_VERSION}_Linux-64bit.tar.gz; \
+	elif [ "$GOARCH"="arm64" ]; then \
+		URL=https://github.com/gohugoio/hugo/releases/download/v${HUGO_VERSION}/hugo_${HUGO_VERSION}_Linux-ARM64.tar.gz; \
+	fi && \
+	echo Downloading ${URL} && \
+	curl -s -L -o hugo.tar.gz "${URL}" && \
+	tar xzvf hugo.tar.gz -C /usr/bin && \
+	rm hugo.tar.gz && \
+	chmod +x /usr/bin/hugo
+
+ENV SHFMT_VERSION=3.2.4
+RUN curl -fsSLo shfmt https://github.com/mvdan/sh/releases/download/v${SHFMT_VERSION}/shfmt_v${SHFMT_VERSION}_linux_$(go env GOARCH) && \
 	chmod +x shfmt && \
 	mv shfmt /usr/bin
+
 RUN curl -sfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh| sh -s -- -b /usr/bin v1.27.0
-RUN GO111MODULE=on go get -tags netgo \
```

Comment:
What's the impact of removing `-tags netgo`?

Reply:
According to https://golang.org/doc/go1.5, `netgo` is no longer necessary.

Reply:
In practice, I haven't noticed any difference.

Reply:
> Thanks Peter for working on this! Another nice test to do could be pushing the image and updating it in CI workflows, so we can test CI running on the new build-image.

I've updated this PR to use newly built image from this PR.